s3_storage_provider.py

```diff
@@ -266,6 +266,9 @@ class _S3Responder(Responder):
     """
 
     def __init__(self):
+        parent_logcontext = current_context()
+        with LoggingContext(parent_context=parent_logcontext):
+            logger.info("Responder For S3")
```

Comment:
what makes you believe this will fix the problem? I don't think it will.

Reply:
I thought the parent context would solve the issue

Reply:
Can you guide me? @richvdh

Reply:
https://github.com/matrix-org/synapse-s3-storage-provider/issues/56#issuecomment-1103873051 has some thoughts